attbillsplitter/main.py

```diff
@@ -141,16 +141,35 @@ def login(self):
             return True
         else:
             if 'promo' in login_submit.url.lower():
-                print ('\U0001F534  Popup window detected during login. '
+                reject_status = self.clickSkipPromo()
+
+                if not reject_status:
+                    print ('\U0001F534  Popup window detected during login. '
                        'Please login you account in a browser and click '
                        'through. Log out your account before you retry. '
                        '(Sometimes you might have to do this multiple times.')
-                return False
+
+                return reject_status
```

Comment:
we can simply `return True` here as for it to get here, `reject_status` has to be `True`.

Reply:
Because there's no return after line 148 inside the if statement. Otherwise put a return false at 149?